src/python/dxpy/cli/dataset_utilities.py

```diff
@@ -1062,6 +1088,15 @@ def extract_assay_germline(args):
                 locus_id_refs = raw_api_call(resp, ref_payload)
                 update_genotype_only_ref(ordered_results, locus_id_refs)
 
+            if args.infer_ref or args.infer_nocall:
+                samples = retrieve_samples(resp, project, selected_assay_name, selected_assay_id)
```

Comment:
`selected_assay_name` is extra here, right? Method definition only takes 3 parameters, and when I run `dx extract_assay germline` from this branch, I get the following error:

```
samples = retrieve_samples(resp, project, selected_assay_name, selected_assay_id)
TypeError: retrieve_samples() takes 3 positional arguments but 4 were given
```